### PR TITLE
chore: update Consumer

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

- [x] added Consumer description;
- [x] update Consumer Creator's fields

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/pages/Consumer/locales/zh-CN.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ export default {
   'page.consumer.form.itemExtraMessage.username': '名称需唯一',
   'page.consumer.notification.warning.enableAuthenticationPlugin': '请启用一种身份认证类插件',
   'page.consumer.username': '名称',
+  'page.consumer.username.required': '请输入消费者名称',
   'page.consumer.updateTime': '更新时间',
-  'page.consumer.list': '消费者列表'
+  'page.consumer.list': '消费者列表',
+  'page.consumer.description': '消费者是路由的消费方，形式包扩开发者、最终用户、API 调用等。创建消费者时，需绑定至少一个认证类插件。',
+  'page.consumer.create-title': '创建消费者',
+  'page.consumer.configure-title': '配置消费者',
```
Comment (liuxiran) [inline web/src/pages/Consumer/locales/zh-CN.ts:28]: May I ask why not keep the same with service use `page.consumer.configure` here?

**Review by liuxiran — APPROVED** [review]

File: web/src/pages/Consumer/locales/zh-CN.ts (hunk 1 above)
Comment (juzhiyuan) [reply web/src/pages/Consumer/locales/zh-CN.ts:28]: ya, when I file the new PR, I use `xxx.configure` without title, this i18n will be handled in a new PR.

**Review by LiteSun — APPROVED** [review]

**Review by membphis — APPROVED** [review]

File: web/src/pages/Consumer/locales/zh-CN.ts (hunk 1 above)
Comment (juzhiyuan) [reply web/src/pages/Consumer/locales/zh-CN.ts]: updated

**Review by nic-chen — APPROVED** [review]

File: web/src/pages/Consumer/locales/zh-CN.ts [hunk 2]
```diff
@@ -20,6 +20,10 @@ export default {
   'page.consumer.form.itemExtraMessage.username': '名称需唯一',
   'page.consumer.notification.warning.enableAuthenticationPlugin': '请启用一种身份认证类插件',
   'page.consumer.username': '名称',
+  'page.consumer.username.required': '请输入消费者名称',
   'page.consumer.updateTime': '更新时间',
-  'page.consumer.list': '消费者列表'
+  'page.consumer.list': '消费者列表',
+  'page.consumer.description': '消费者是路由的消费方，形式包扩开发者、最终用户、API 调用等。创建消费者时，需绑定至少一个认证类插件。',
```
Comment (starsz) [inline web/src/pages/Consumer/locales/zh-CN.ts:26]: typo？
"包扩" => "包括"

**Review by starsz — CHANGES_REQUESTED** [review]

File: web/src/pages/Consumer/locales/zh-CN.ts (hunk 2 above)
Comment (juzhiyuan) [reply web/src/pages/Consumer/locales/zh-CN.ts]: updated

**Review by starsz — APPROVED** [review]

**Pull request merged**